artifacts/examples/istio-canary.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     portName: http
     portDiscovery: true
     gateways:
-    - public-gateway.istio-system.svc.cluster.local
+    - istio-system.svc.cluster.local/public-gateway
```

Comment:
Have you test this on a cluster? According to Istio docs this is not valid, the prefix should be a namespace not a hostname.

Reply:
can you tell me the required tools to test this on a cluster?